webapp/app.py

```diff
@@ -131,3 +132,34 @@ def inject_utilities():
 @app.context_processor
 def inject_today_date():
     return {"current_year": datetime.date.today().year}
+
+
+S3_BUCKET_URL = "https://discourse-charmhub-io.s3.eu-west-2.amazonaws.com"
+
+
+@app.route("/assets")
+def assets():
+    url = request.args.get("url", "")
+    if not url.startswith(S3_BUCKET_URL):
+        abort(404)
+
+    result = requests.get(url)
+    return send_file(
```

Comment:
before we return can we check the mimetype is an image too? The more checks we have here the better. Adding tests to the methods with as many different use-cases as you can think of would also be good.

Reply:
Yeah makes sense, but I'd rather start with a subset and if something isn't working add that mimetype in. I don't think we need to come up with a comprehensive list off the bat